src/main/java/com/sprint/mission/discodeit/controller/UserController.java

```diff
@@ -0,0 +1,72 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.UserCreateRequest;
+import com.sprint.mission.discodeit.dto.UserDTO;
+import com.sprint.mission.discodeit.dto.UserUpdateRequest;
+import com.sprint.mission.discodeit.dto.UserStatusUpdateRequest;
+
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.entity.UserStatus;
+import com.sprint.mission.discodeit.service.UserService;
+import com.sprint.mission.discodeit.service.UserStatusService;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/users")
+@RequiredArgsConstructor
+public class UserController {
+
+  private final UserService userService;
+  private final UserStatusService userStatusService;
+
+  @PostMapping(value = "", consumes = {MediaType.MULTIPART_FORM_DATA_VALUE})
+  public ResponseEntity<UserDTO> createUser(
+      @RequestPart("userCreateRequest") UserCreateRequest userCreateRequest,
+      @RequestPart(value = "profile", required = false) MultipartFile profile
+  ) {
+    User user = userService.create(userCreateRequest, profile);
+    return ResponseEntity.status(HttpStatus.CREATED).body(UserDTO.createEntity(user));
+  }
+
+  @PutMapping(value = "/{userId}", consumes = {MediaType.MULTIPART_FORM_DATA_VALUE})
+  public ResponseEntity<UserDTO> updateUser(
+      @PathVariable UUID userId,
```

Comment:
이렇게 코드를 수정해야 정상동작하는 것 같은데요,  
이부분 한번 확인가능하실까요?  

```suggestion
      @PathVariable("userId") UUID userId,
```

기존 코드로는 아래와 같은 에러가 발생하더라구요.

```
Name for argument of type [java.util.UUID] not specified, and parameter name information not available via reflection. Ensure that the compiler uses the '-parameters' flag.
```